External/FEXCore/Source/Interface/Core/ArchHelpers/Arm64.cpp

```diff
@@ -1559,6 +1565,53 @@ bool HandleAtomicLoad128(void *_ucontext, void *_info, uint32_t Instr) {
   return true;
 }
 
+static uint64_t HandleCAS_NoAtomics(void *_ucontext, void *_info)
+{
+  mcontext_t* mcontext = &reinterpret_cast<ucontext_t*>(_ucontext)->uc_mcontext;
+  siginfo_t* info = reinterpret_cast<siginfo_t*>(_info);
```

Comment:
Please add a check here just like `HandleCASAL` to ensure the si_code is `BUS_ADRALN` otherwise we could get spurious checks on invalid memory accesses

Reply:
That would duplicate the check made in the only caller of this function `HandleAtomicLoadstoreExclusive`, is that alright?

Reply:
Oh no, you're right. I forgot that the caller also did the check. Thanks!